### PR TITLE
viewport

### PR DESCRIPTION
**Pull request by xiaoxiao921**

### CODE REVIEW

**Comment by xiaoxiao921** [comment]
https://github.com/citizenfx/fivem/blob/1749be021ce2e3b9cf3fdc91d480399c8c0ce0e6/code/components/extra-natives-five/include/GamePrimitives.h#L115

**Review by gir489returns — APPROVED** [review]
FiveM totally didn't steal this from the leaked source or PDB, lmao.

**Comment by xiaoxiao921** [comment]
Yeah didn't realize but that's someone from fiveM pring it two weeks ago, so could be from leak...

**Comment by gir489returns** [comment]
> Yeah didn't realize but that's someone from fiveM pring it two weeks ago, so could be from leak...

I mean, they work for Rockstar now, so someone from the FiveM team who works at Rockstar could've also updated it, so who knows.

File: graphics/CViewport.hpp [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+struct CViewportGame
+{
+public:
+    virtual ~CViewportGame() = 0;
+
+private:
+    char m_pad[8];
+
+public:
+    grcViewport viewport;
```
Comment (Yimura) [inline graphics/CViewport.hpp:13]: `CViewportGame isn't part of the `rage` namespace, how is this able to compile? I'm guessing the headers haven't been added to the cpp test compile file...

**Review by Yimura — CHANGES_REQUESTED** [review]

**Review by Yimura — APPROVED** [review]

**Pull request merged**